src/components/Layout.js

```diff
@@ -0,0 +1,19 @@
+import Header from './Header/Header';
+import NavBar from './NavBar/NavBar';
+
+export default function Layout({ children }) {
+	// styles the main html tag
+	const styles = {
+		margin: '9vh 0 0 0 ',
+		display: 'flex',
+		flexDirection: 'column',
+	};
+
```

Comment:
As your styles-variable is fixed without any variants, you could build a Main.styled.js with those attributes to keep this component clean.

Reply:
Done